api/http/api_deployments.go

```diff
@@ -402,42 +409,67 @@ func (d *DeploymentsApiHandlers) GenerateImage(w rest.ResponseWriter, r *rest.Re
 }
 
 // ParseMultipart parses multipart/form-data message.
-func (d *DeploymentsApiHandlers) ParseMultipart(r *rest.Request) (*model.MultipartUploadMsg, error) {
-	multipartUploadMsg := &model.MultipartUploadMsg{
+func (d *DeploymentsApiHandlers) ParseMultipart(r *multipart.Reader) (*model.MultipartUploadMsg, error) {
+
+	uploadMsg := &model.MultipartUploadMsg{
 		MetaConstructor: &model.SoftwareImageMetaConstructor{},
 	}
-	multipartUploadMsg.MetaConstructor.Description = r.FormValue("description")
-
-	sizeValue := r.FormValue("size")
-	var size int64
-	var err error
-	if sizeValue != "" {
-		size, err = strconv.ParseInt(sizeValue, 10, 64)
+	var sizeParsed bool
+	for {
+		part, err := r.NextPart()
```

Comment:
I have never done that in go, so please excuse me asking: so we do nothing with the `part` thing?